keyboards/handwired/nozbe_macro/keymaps/default/keymap.c

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2021 Marcin Leon Omelan (@rozPierog)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum layer_names { _BASE, _FN };
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(LT(_FN, KC_N), LALT(KC_UP), LALT(KC_DOWN), LALT(KC_D)),
+    [_FN]   = LAYOUT(KC_TRNS, KC_1, KC_2, BL_STEP),
```

Comment:
This is more of a nitpick for readability, but: 
```suggestion
enum layer_names { 
    _BASE, 
    _FN,
};

const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [_BASE] = LAYOUT(
        LT(_FN, KC_N), LALT(KC_UP), LALT(KC_DOWN), LALT(KC_D)
    ),
    [_FN]   = LAYOUT(
        KC_TRNS, KC_1, KC_2, BL_STEP
    ),
```